res/skins/Tango/skin.xml

```diff
@@ -26,8 +26,9 @@
   <manifest>
     <title>Tango</title>
     <author>ronso0@mixxx.org</author>
-    <version>2.3.0</version>
-    <description>A controller-focused nighttime skin. Almost all controls can be hidden with skin menu.</description>
+    <version>2.3.0.01</version>
+    <description>A very flexible, controller-focused nighttime skin with stacked waveforms,
```

Comment:
I don't like this description of a skin as "controller-focused". It misleadingly implies that it can only be used with a controller.

Reply:
okay, I'll remove it.

Reply:
what about this, that's the USP I had in mind when designing it ;)
"A flexible nighttime skin with stacked waveforms, 4 decks and
8 samplers that can be minimized to the bare essentials."

Reply:
Yup, sounds good to me.